include/bluetooth/mesh/light_ctrl_cli.rst

```diff
@@ -0,0 +1,29 @@
+.. _bt_mesh_light_ctrl_cli_readme:
+
+Light Lightness Control Client
+##############################
+
+The Light Lightness Control (LC) Client configures and interacts with the Light LC Server.
+
+Contrary to the Server model, the Client only creates a single model instance in the mesh composition data.
+The Light LC Client may send messages to both the Light LC Server and the Light LC Setup Server, as long as it has the right application keys.
+
+Extended models
+===============
```

Comment:
Please make sure all headings follow the same heading formatting, as per https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/documentation/index.html#doc-guidelines

So this should be marked with `*`, then we have `=` and `-`.